cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -215,3 +218,36 @@ func (b *Builder) Do(ctx context.Context) (runtime.Object, error) {
 
 	return bImpl.build(ctx)
 }
+
+func (b *Builder) buildPodSpec(templateSpec corev1.PodSpec) corev1.PodSpec {
+	bundle := mergeBundles(b.volumeBundles)
+
+	templateSpec.Volumes = append(templateSpec.Volumes, bundle.Spec.Volumes...)
+	for i := range templateSpec.Containers {
```

Comment:
volume mounts should also be added to initContainers.

But the command should only be added to the first primary container. Requests too.

And this has to be documented somewhere, probably as part of the APIs.

Reply:
Done.